torch/fx/experimental/symbolic_shapes.py

```diff
@@ -342,6 +353,8 @@ def unary_magic_impl(self):
             pytype = int
         elif method in always_float_magic_methods:
             pytype = float
+        elif method in ["sym_int"]:
+            pytype = int
```

Comment:
This is not consistent with the new organization, which is that we have a few top level method lists for matching these. Probably put this in `always_int_magic_methods`

Reply:
fixed.